File: Day28/string_case_conversion.c
#include<stdio.h>
#include<ctype.h>

int main(){
    
    char str[100];
    printf("Enter a String : ");
    fgets(str , sizeof(str), stdin); 

    printf("1. Upper Case to Lower case. \n");
    printf("2. Lower case to Upper case. \n");
    printf("3. Toggle case. \n ");
    int choice;
    scanf("%d",&choice);

    switch(choice){
        case 1:
            for (int i = 0; i < str[i]; i++){
                str[i] = tolower(str[i]);
            }
            printf("Lower case String : %s " , str);
            break;
        case 2:
            for (int i = 0; i < str[i]; i++){
                str[i] = toupper(str[i]);
            }
            printf("Upper case String : %s " , str);
            break;
        case 3:
            for (int i = 0; i < str[i]; i++){
                if(isupper(str[i])){
                    str[i] = tolower(str[i]);
                }else{
                    str[i] = toupper(str[i]);
                }
            }
            printf("Toggled case String : %s " , str);
            break;
        default:
            printf("Invalid choice  ");
    }
    return 0;
}

File: Day28/string_case_conversion.py
"""Function for String Conversion """
def string_case_conversion():

    """Get the input string from the user"""
    input_str = input("Enter a String : ")

    print("1. Upper Case to Lower case. ")
    print("2. Lower case to Upper case. ")
    print("3. Toggle case. ")

    choice = int(input("Enter your choice: "))


    if choice == 1:
        print("Lower case String : " , input_str.lower())
    elif choice == 2:
        print("Upper case String : " , input_str.upper())
    elif choice == 3:
        print("Toggled case String : " , input_str.swapcase())
    else:
        print("Invalid choice  ")

string_case_conversion()

